src/app.jsx

```diff
@@ -179,6 +181,11 @@ class App extends React.Component {
     this.setState(newState);
   }
 
+  getMonth() {
+    let date = this.state.timelineDate;
+    return date.split('-')[1];
```

Comment:
moment has a method to get the month. As we are already loading moment here, you can use it.